client/src/__tests__/unit-tests/language/EmbeddedLanguageDocsManager.test.ts

```diff
@@ -0,0 +1,22 @@
+/* --------------------------------------------------------------------------------------------
+ * Copyright (c) 2023 Savoir-faire Linux. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for license information.
+ * ------------------------------------------------------------------------------------------ */
+
+import fs from 'fs'
+import path from 'path'
+import { embeddedLanguageDocsManager } from '../../../language/EmbeddedLanguageDocsManager'
+
+describe('EmbeddedLanguageDocsManager', () => {
+  it('can create and delete embedded language documents folder', async () => {
+    const storagePath = path.join(__dirname, 'generated-content')
+    await embeddedLanguageDocsManager.setStoragePath(storagePath)
+    expect(embeddedLanguageDocsManager.embeddedLanguageDocsFolder).not.toBeUndefined()
```

Comment:
toBeDefined to avoid double negation?

Reply:
Done.

Funny, we used this in many other places. Guess it just seems natural, since we do `!== undefined` all the time.